src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -1563,6 +1563,11 @@ void runtime::gpu::GPU_Emitter::emit_Subtract(EMIT_ARGS)
 }
 
 void runtime::gpu::GPU_Emitter::emit_Sum(EMIT_ARGS)
+{
+    runtime::gpu::GPU_Emitter::emit_Sum_1(external_function, writer, node, args, out);
+}
+
+void runtime::gpu::GPU_Emitter::emit_Sum_0(EMIT_ARGS)
```

Comment:
Comments for something ambiguously named are mandatory. What does this function do? Same comment for Sum_1
Also in comment, why this is better or worse or whatever than other functions. What does this function bring to the table?

Reply:
Done

Reply:
Done